OpenRA.Game/Traits/Player/PlayerResources.cs

```diff
@@ -37,7 +43,10 @@ IEnumerable<LobbyOption> ILobbyOptions.LobbyOptions(Ruleset rules)
 			var startingCash = SelectableCash.ToDictionary(c => c.ToString(), c => "$" + c.ToString());
 
 			if (startingCash.Any())
-				yield return new LobbyOption("startingcash", "Starting Cash", new ReadOnlyDictionary<string, string>(startingCash), DefaultCash.ToString(), DefaultCashLocked);
+				yield return new LobbyOption("startingcash", "Starting Cash", "Change the amount of cash that you start with",
```

Comment:
Maybe move those descriptions to .yaml too. Not sure about others, but for crates at least it says `random bonuses or penalties` but some mods may have only bonuses for example. Or just wanna change the wording a bit.

Reply:
Ok, done.